sdk/communication/communication-sms/test/smsClient.mocked.spec.ts

```diff
@@ -19,11 +19,14 @@ import { apiVersion } from "../src/generated/src/models/parameters";
 import { SmsClient, SmsSendRequest } from "../src/smsClient";
 
 const API_VERSION = apiVersion.mapper.defaultValue;
+const TEST_NUMBER = "+18005551234";
 
 describe("[mocked] SmsClient", async () => {
   const baseUri = "https://contoso.api.fake:443";
+  const connectionString = `endpoint=${baseUri};accesskey=banana`;
```

Comment:
is it banana normally used? or can we change it for something else like 'accessKeyCredential'?

Reply:
yes, this "banana" key is used in our other mocked tests in the other comms sdks. This is a mocked test so I don't think this is an issue